data/upgrades/device.json

```diff
@@ -82,7 +82,7 @@
         "device": {
           "name": "Proximity Mine",
           "type": "Mine",
-          "effect": "After a ship overlaps or moves through this device, it detonates. When this device detonates, that ship rolls 2 attack dice. That ship then suffers 1 [Hit]/[Critical Hit] damage for each matching result."
+          "effect": "After a ship overlaps or moves through this device, it detonates. When this device detonates, that ship rolls 2 attack dice. That ship then suffers 1 [Hit] plus 1 [Hit]/[Critical Hit] damage for each matching result."
```

Comment:
I think this should read:

```suggestion
          "effect": "After a ship overlaps or moves through this device, it detonates. When this device detonates, that ship rolls 2 attack dice. That ship then suffers 1 [Hit] damage plus 1 [Hit]/[Critical Hit] damage for each matching result."
```

Reply:
But if I add that extra "damage" text then it won't exactly match the errata text. Notice in the snippet below that "damage" is only written once.

![proxmine](https://user-images.githubusercontent.com/20940667/47514764-03e56900-d836-11e8-8deb-e06545574a7b.PNG)

Reply:
Ah, I was looking at this part:

![proximity-mines](https://user-images.githubusercontent.com/834902/47529760-fae1af80-d8a8-11e8-96e2-8e227125fe2a.png)

Seems like FFG isn't consistent in the same document ;)